### PR TITLE
feat: add logout functionality for Header/burger menu

### PR DESCRIPTION
**Pull request by ElijahWolf1986**

### CODE REVIEW

File: .env.example
Comment (algoritmi4) [inline .env.example:1]: dont change .env.example file

File: src/features/burgerMenu/ui/burgerMenu.tsx [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+import { LogoutButton } from "@/features/authentication/logout";
 
 export function BurgerMenu() {
+  const refresh = selectRefreshToken();
+
   return (
-    <Menu.Items className="absolute flex flex-col right-0 z-[11] mt-10 w-[293px] h-[435px] px-10 py-[30px] rounded-[20px] bg-white shadow justify-start items-start gap-[15px] inline-flex focus:outline-none">
-      {menuItems.map(({ img, name }, id) => {
+    <Menu.Items className="absolute flex flex-col right-0 z-[11] mt-10 w-[293px] h-[410px] px-10 py-[30px] rounded-[20px] bg-white shadow justify-start items-start gap-[15px] inline-flex focus:outline-none">
+      {menuItems.map(({ img, name, link }, id) => {
```
Comment (algoritmi4) [inline src/features/burgerMenu/ui/burgerMenu.tsx]: You can reduce this code like
`{menuItems.map(({ img, name, link }, id) => (<Menu.Item>...<Menu.Item>)}` - that reads better.

File: src/features/burgerMenu/ui/burgerMenu.tsx [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
         return (
           <Menu.Item key={id}>
-            {() => <MenuItem img={img} name={name} />}
+            {() => <MenuItem img={img} name={name} link={link} />}
```
Comment (algoritmi4) [inline src/features/burgerMenu/ui/burgerMenu.tsx]: Why do you return MenuItem from callback? I think that reason why you have console warning.
Also, a good solution would be to put the map function into a const. For example, you can check my realization of features/eventChoice/ui/EventSlider.

File: src/features/burgerMenu/ui/burgerMenu.tsx (hunk 1 above)
Comment (ElijahWolf1986) [reply src/features/burgerMenu/ui/burgerMenu.tsx]: fixed

File: src/features/burgerMenu/ui/burgerMenu.tsx (hunk 2 above)
Comment (ElijahWolf1986) [reply src/features/burgerMenu/ui/burgerMenu.tsx]: fixed

**Review by algoritmi4 — APPROVED** [review]

**Pull request merged**